### PR TITLE
Array sub

### PR DESCRIPTION
**Pull request by JayantGoel001**
### Have you read the [Contributing Guidelines on Pull Requests](https://github.com/TesseractCoding/NeoAlgo/blob/master/CONTRIBUTING.md#pull-requests)?

(Yes.)

### Description

(Given an array and an integer k, find the maximum for each and every contiguous subarray of size k.)

### Checklist

- [x] I've read the contribution guidelines.
- [x] I've checked the issue list before deciding what to submit.
- [x] I've edited the `README.md` and link to my code.

## Related Issues or Pull Requests

#1425

### CODE REVIEW

File: C-Plus-Plus/cp/ARRAYSUB.cpp [hunk 1]
```diff
@@ -0,0 +1,52 @@
+#include<iostream>
```
Comment (iamrajiv) [inline C-Plus-Plus/cp/ARRAYSUB.cpp:4]: @JayantGoel001 Add a little bit description of the problem and link of the problem.

**Review by iamrajiv — CHANGES_REQUESTED** [review]

**Review by iamrajiv — APPROVED** [review]

**Review by atarax665 — APPROVED** [review]
LGTM :+1:

**Comment by iamrajiv** [comment]
@all-contributors please add @JayantGoel001 for code

**Comment by iamrajiv** [comment]
@all-contributors please add @JayantGoel001 for code

**Comment by iamrajiv** [comment]
@all-contributors please add @JayantGoel001 for code

**Comment by iamrajiv** [comment]
@all-contributors please add @JayantGoel001 for code

**Pull request merged**